src/Streamly/Data/Unfold.hs

```diff
@@ -34,13 +34,13 @@
 -- @Unfold m a b@ can be considered roughly equivalent to an action @a -> t m
 -- b@ (where @t@ is a stream type). Instead of using an 'Unfold' one could just
 -- use a function of the shape @a -> t m b@. However, working with stream types
--- like 'S.SerialT' does not allow the compiler to perform stream fusion
+-- like t'Streamly.SerialT' does not allow the compiler to perform stream fusion
```

Comment:
Without annotating with `t` haddock tries to create a link to a function `SerialT`. This looks like this should be fixed in recent version of haddock though it doesn't work for me.